src/main/java/Baymax.java

```diff
@@ -0,0 +1,92 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Baymax {
+    public static void main(String[] args) {
+        String name = "Baymax";
```

Comment:
this should be a constant so can consider making it all caps and making it static and final